core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/dotnet/DotNetCoordinates.java

```diff
@@ -37,6 +37,11 @@ public DotNetCoordinates(String packageId, String version) {
     @Override
     public String getVersion() { return version; }
 
+    @Override
+    public String getPurl() {
+        return String.format("pkg:nuget/%s@%s", getName(), getVersion());
```

Comment:
In general, purl-components need to be percent-encoded, see

https://github.com/heremaps/oss-review-toolkit/blob/673c9b817cb71fed1bcd313d51d4b7788fbd2e61/model/src/main/kotlin/Identifier.kt#L126
https://github.com/package-url/purl-spec#rules-for-each-purl-component

Reply:
Also, all implementations except the Maven one seem to miss the "namespace" part.

Reply:
I added this in my latest update. Concerning the namespace part, I think this is a minor bug in ORT, if I compare the examples from https://github.com/package-url/purl-spec:

pkg:npm/%40angular/animation@12.3.1
pkg:npm/foobar@12.3.1

pkg:nuget/EnterpriseLibrary.Common@6.0.1304

Reply:
Oh, you're right, "The name is prefixed by a '/' separator when the namespace is not empty"!